.editorconfig

```diff
@@ -0,0 +1,163 @@
+# editorconfig.org

```

Comment:
is this file needed?

Reply:
It silently enforces nF coding guidelines, without having to think about it. It also does not interfere with different guidelines in other projects that use .editorconfig. Also, I do not need to import any settings.

So yeah, I think this file is indeed needed very much.